client/my-sites/theme/main.jsx

```diff
@@ -145,6 +145,14 @@ class ThemeSheet extends Component {
 		return !! this.props.screenshots;
 	};
 
+	isRemoved = () => {
+		// If a theme has been removed by a theme shop, then the theme will still exist and a8c will take over any support responsibilities.
+		return (
+			this.props?.taxonomies?.theme_status.filter( ( status ) => status.slug === 'removed' )
+				.length === 1
+		);
+	};
+
```

Comment:
Sorry for this comment, I know this isn't RfR just yet. As per the `propType` spec, it seems that the `taxonomies` key will always exist. I also think this function could be a bit simpler, would this suggestion have the same result as the one you're expecting?

```suggestion
// If a theme has been removed by a theme shop, then the theme will still exist and a8c will take over any support responsibilities.
	isRemoved = () =>
		!! this.props.taxonomies?.theme_status.find( ( status ) => status.slug === 'removed' );
```

Reply:
Don't apologise, that's a helpful comment, thanks!